plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -322,7 +322,7 @@ mod tests {
     #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
     #[serial_test::serial]
     async fn test_message_dispatch() -> anyhow::Result<()> {
-        let test_config_path = "/some/test/config";
+        let test_config_path = "/tmp"; //Pass some existing path
```

Comment:
is this needed?

Reply:
Yes, to add a watcher to any directory, that directory should exists.

Reply:
I'm just confused as to why a change was needed in the c8y configuration plugin? :smile: